dependency-check-suppressions.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
 
+    <suppress>
+        <notes>Transitive dependency not used by our source. However it is needed for run time.</notes>
```

Comment:
can you please check with Mutlu whether is ok to add this suppression? he's the specialist in this area :)